packages/api/src/schema/schema.yaml

```diff
@@ -1044,6 +1044,10 @@ components:
           example: ea03f37e-f861-4cdd-b495-0e60b6d753ad
           index: true
           description: Used to form recording URL for HLS playback
+        generateMp4s:
```

Comment:
I think an "action" field like this makes sense in the task objects, but not in the assets which are a description of an entity, not of an action. 

In this case, WDYT of using a descriptive field instead of an action here? I like `staticMp4s` instead, WDYT? 

FTR mux uses a `mp4_support: standard` field instead (not even a boolean). I think we're fine with a bool for now.